app/assets/scripts/components/table.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import T from 'prop-types';
+
+/*
+ * Table component
+ * @param data - data object used to construct table
+ *    expected in the following format:
+ *    {
+ *      header1: {
+ *        values: [Array](required),
+ *        formatCell: [function](optional),
+ *        formatHeader: [function])(optional),
+ *        sortable: [boolean](optional, default false),
+ *        cellWidth: [string](optional)
+ *      },
+ *      ...,
+ *      headerN: {
+ *        values: [Array](required),
+ *        formatCell: [function](optional),
+ *        formatHeader: [function])(optional),
+ *        sortable: [boolean](optional, default false),
+ *        cellWidth: [string](optional)
+ *      }
+ *    }
+ */
+const Table = styled.table``;
+const Row = styled.tr`
+  background-color: ${({ index }) => (index % 2 === 0 ? '#ffffff' : '#f4f9fe')};
```

Comment:
Should we be referencing a theme for these colors so we don't have to hunt them down in later component development?

Reply:
yes i agree that using explicit colors here is not the best choice. what would be ideal is if we can set up a theme to parse out variables from the sass files. i think lightest fix for this is to use class name and set styles with sass variables. what do you think? @necoline @AliceR